frontend/src/components/Documentation.vue

```diff
@@ -157,8 +157,8 @@
           Users can also hover a node to view its name/id or left-clik on a node (once the graph has stopped moving) to display some of its information in the sidebar. Additonal information on the corresponding selected element can be accessed by clicking the <i>GEM Browser</i> button.<br>
 
           <h5 id="Data-overlay" class="is-size-5">Data overlay</h5>
-          <div class="columns is-marginless">
-            <div class="column is-paddingless is-8 content">
+          <div class="columns m-0">
+            <div class="column px-0 py-0 is-8 content">
```

Comment:
Could use `p-0` here.

Reply:
Sorry about this, I'm not sure how I ended up with `px-0 and py-0` since I've used `p-0` in other places.